scripts/changelog.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

so that `nix` users could also use it